Makefile

```diff
@@ -36,7 +35,6 @@ envinit:
 #  - add to $PATH: /usr/local/opt/go@1.16/bin
 osx-init:
 	/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
-	brew install yarn
```

Comment:
brew install npm instead?

Reply:
no, node already ships npm but maybe not always the latest greatest.